app/controllers/api/v1/admin/articles_controller.rb

```diff
@@ -13,15 +17,8 @@ def create
 
   private
 
-  def attach_image(article)
-    params_image = params[:article][:image]
-    if params_image.present?
-      DecodeService.attach_image(params_image, article.image)
-    end
-  end
-
   def article_params
-    params.require(:article).permit(:title, :teaser, :content, :category, :premium)
+    params.require(:article).permit(:title, :teaser, :content, :category, :premium, :location)
```

Comment:
should location be required? Then you can't create an article without location?

Reply:
it is required. Check the model out